app/services/explorer.js

```diff
@@ -1032,38 +1063,54 @@ export default Ember.Service.extend({
      * @return {Ember.RSVP.Promise<Array<RiakNode>>|Array<RiakNode>}
      */
     getNodesForCluster(cluster, store) {
-        if(Ember.isEmpty(cluster.get('riakNodes'))) {
-            return store.query('riak-node', { clusterId: cluster.get('id') })
+        if(Ember.isEmpty(cluster.get('nodes'))) {
+            return store.query('node', { clusterId: cluster.get('id') })
                 .then(function(nodes) {
-                    cluster.set('riakNodes', nodes);
+                    cluster.set('nodes', nodes);
 
                     return nodes;
                 });
         } else {
-            return cluster.get('riakNodes');
+            return cluster.get('nodes');
         }
     },
 
     /**
-     * Returns the results of a GET Node Stats HTTP call.
+     * Gets and sets the node stats property. Returns the node model object.
      *
      * @method getNodeStats
-     * @param {String} nodeId
-     * @return {Ember.RSVP.Promise<Array<Hash>>}
+     * @param {DS.Model} Node
+     * @return {Ember.RSVP.Promise<Node>}
      */
-    getNodeStats(nodeId) {
-        var propsUrl = `${this.apiURL}riak/nodes/${nodeId}/stats`;
-        var propsResult = Ember.$.ajax( propsUrl, { dataType: "json" } );
-
-        return propsResult.then(
-            function(data) {
-                var statsArray = objectToArray(data);
-                return {
-                    node: nodeId,
-                    stats: statsArray
-                };
-            }
-        );
+    getNodeStats(node) {
+        let url = `${this.apiURL}riak/nodes/${node.get('id')}/stats`;
+
+        return new Ember.RSVP.Promise(function(resolve, reject) {
+            let request = Ember.$.ajax({
+                url: url,
+                type: 'GET'
+            });
+
+            request.done(function(data) {
+                let statsArray = objectToArray(data).sort(function(a, b) {
+                    if (a.key < b.key) {
+                        return -1;
+                    } else if (a.key > b.key) {
+                        return 1;
+                    } else {
+                        return 0;
+                    }
+                });
```

Comment:
I went ahead and did alpha-sort on both node stats and config. If stats should not be, let me know and I will reverse.